recipes/demjson3/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "demjson3" %}
+{% set version = "3.0.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ab9aabdd85695f3684fc296f39766a2730f6c8de81d23f7048073dfe2f616d80
+
+build:
+  number: 0
+  noarch: python
```

Comment:
I see. `demjson3` is a fork of `demjson` which in `setup.py` has a key `scripts`: https://github.com/dmeranda/demjson/blob/master/setup.py#L53
Nonetheless, the conda-forge package recipe for `demjson` is noarch: 
https://github.com/conda-forge/demjson-feedstock/blob/master/recipe/meta.yaml#L13
so I adapted its recipe for `demjson3`.
If this is anyway a issue, can you please elaborate a bit more about the preferred solution? `scripts` in upstream have to be `entry points`? Thanks

Reply:
Yes, that recipe is wrong, it shouldn't be noarch. Can you make an issue there, please?

`entry points` would be the preferred approach, yes. It would be grat if you can make a PR with that and they release a new version. Otherwise, you can also include the PR as a patch here.
If you don't know how to patch this you can also just remove the noarch here.

More info on patching: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#patches

Reply:
I made a PR to upstream and patched the recipe, maintaining noarch. Ready for re-reviewing, thanks for your help